CHANGELOG.md

```diff
@@ -21,6 +21,11 @@
   function parameter can be replaced with an opaque `some` type.  
   [SimplyDanny](https://github.com/SimplyDanny)
 
+* Add `excluded` configuration option to `unneeded_override` to opt out checking methods with a given name.
```

Comment:
Not sure if `excluded` is descriptive enough. Likely it is. Could be `excluded_methods` as well.